samples/boards/altera_max10/sysid/src/main.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include "altera_avalon_sysid.h"
+
+void main(void)
+{
+	s32_t sysid;
+
+	sysid = alt_avalon_sysid_test();
+	if (!sysid)
```

Comment:
zephyr coding style requires the use of braces at all times

Reply:
oops!! sorry i missed it.